### DISCUSSION TITLE
Caching only polygon data using geomap

### DISCUSSION THREAD

**Discussion by JSher1** [wet-boew/wet-boew#9029] [General]
We have a table of 400 polygon areas with data for each of them.  The polygon data is 4mb while the data that changes is only 150k.

We have attempted building the map using both kml and html in an attempt to cache just the polygon data but have been unsuccessful.

The polygons must be clickable and highlight when clicked showing the data from the table.  

Any ideas would be appreciated.

**Comment by jvanulde** [comment]
@JSher1 did you find a workaround for this?

**Reply by JSher1** [reply]
No sorry, we moved away from this I believe
